### PR TITLE
add pacing_type_temporary property and change uses of course_run.paci…

### PR DESCRIPTION
**Pull request by michaelroytman**
…ng_type to use it instead, where appropriate

### CODE REVIEW

**Comment by michaelroytman** [comment]
@edx/educator-dahlia Please review. Thanks!

File: course_discovery/apps/publisher/models.py [hunk 1]
```diff
@@ -435,6 +435,27 @@ def paid_seats(self):
...
+    @property
+    def pacing_type_temporary(self):
+        """
+            This property serves as a temporary intemediary in order to support a waffle
+            switch that will toggle between the original database backed pacing_type value
+            and a new read-only value that is pulled from course_discovery.
+
+            Once the switch is no longer needed, the pacing_type field will be removed,
+            and this property will be renamed appropriately.
+        """
```
Comment (fysheets) [inline course_discovery/apps/publisher/models.py]: 👍 Thank you for the detailed comment

**Comment by michaelroytman** [comment]
Just as an FYI, @bderusha.

**Review by fysheets — COMMENTED** [review]
This looks good, but looking at the project structure - I noticed there were references to `pacing_type` in the `publisher_comments` section as well. Wanted to sanity check if we should update those as well?

**Comment by michaelroytman** [comment]
@fysheets Yes, you're right. I'll fix that now.

**Comment by michaelroytman** [comment]
@fysheets This should be all set for another look. Thanks!

**Review by fysheets — APPROVED** [review]
Thanks!

**Pull request merged**